cmd/opampsupervisor/supervisor/supervisor.go

```diff
@@ -534,7 +545,7 @@ func (s *Supervisor) getBootstrapInfo() (err error) {
 	case err = <-done:
 		if errors.Is(err, errNonMatchingInstanceUID) {
 			// try to report the issue to the OpAMP server
-			if startOpAMPErr := s.startOpAMPClient(); startOpAMPErr == nil {
+			if startOpAMPErr := s.startOpAMPClient(context.Background()); startOpAMPErr == nil {
```

Comment:
can you not pass the ctx of the outside span from line 403?

Reply:
good catch, thank you -  seems like i missed that one - I'll try it out to be sure, but I think the context from further above should be the correct one to use here